client/src/partials/price_list/pricelist.html

```diff
@@ -1,195 +1,202 @@
-<header data-header>
-  {{ "PRICE_LIST.TITLE" | translate }}
-</header>
-
-<nav>
-  <div class="pull-left">
-    <ol class="breadcrumb">
-      <li><a href="#/"><span class="glyphicon glyphicon-home"></span></a></li>
+<div class="flex-header">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static">{{ "TREE.ADMIN" | translate }}</li>
       <li class="active">{{ "PRICE_LIST.TITLE" | translate }}</li>
     </ol>
   </div>
+</div>
 
-  <div class="pull-right">
-    <button class="btn btn-sm btn-default" ng-click="addList()">
-      <span class="glyphicon glyphicon-plus-sign"></span> {{ "PRICE_LIST.NEW" | translate }}
-    </button>
-  </div>
-</nav>
-
-<main>
-  <div class="row">
-    <div class="col-xs-4">
-      <div class="panel panel-default">
-        <div class="panel-heading">
-          <span>{{ "PRICE_LIST.LISTS" | translate }}</span>
+<div class="flex-util">
+  <button class="btn btn-sm btn-default" id="create" ng-click="PriceListCtrl.create()" data-method="create">
+    <span class="glyphicon glyphicon-plus-sign"></span> {{ "PRICE_LIST.NEW" | translate }}
+  </button>
+</div>
+
+
+<div class="flex-content">
+  <div class="container-fluid">
+    <div class="row">
+
+      <div class="col-xs-6">
+        <div class="panel panel-default" style="overflow: auto; max-height: 500px;">
+          <div class="panel-heading">
+            {{ "PRICE_LIST.LISTS" | translate }}
+          </div>
+
+          <table class="table table-condensed">
+            <thead>
+              <tr>
+                <th>{{ "COLUMNS.DESCRIPTION" | translate }}</th>
+                <th colspan="3" style="width: 20%;">{{ "COLUMNS.ACTION" | translate }}</th>
```

Comment:
All these colspans do not line up.  I haven't looked at the rendered page, but it looks like you have four columns here,
three in the `<tbody>` and only two if there is no data.  Can you verify that this renders properly?